### PR TITLE
skip test_positive_download_tailoring_file test

### PR DESCRIPTION
**Pull request by jameerpathan111**
Skip test_positive_download_tailoring_file because of [bz#1857572](https://bugzilla.redhat.com/show_bug.cgi?id=1857572 )
and update test_negative_create_with_invalid_file.

### CODE REVIEW

File: tests/foreman/ui/test_oscappolicy.py [hunk 1]
```diff
@@ -125,7 +125,7 @@ def test_positive_check_dashboard(
         )
         policy_details = session.oscappolicy.details(name)
         assert policy_details['HostsBreakdownStatus']['total_count'] == 1
-        assert policy_details['HostBreakdownChart']['hosts_breakdown'] == '100%'
+        assert policy_details['HostBreakdownChart']['hosts_breakdown'] == '100%Not audited'
```
Comment (mirekdlugosz) [inline tests/foreman/ui/test_oscappolicy.py:128]: What's up with that "Not audited"?
Seems similar to change we had recently in another PR: https://github.com/SatelliteQE/robottelo/pull/7845#discussion_r443466606

For me, the key question is: is this something we explicitly want to check, or is it something that just happened to appear and we want to see test still passing?

If that's second option, I would recommend checking Airgun if it's possible to put this new data in it's own widget. If it's not feasible, I would just drop this part of string before assertion.

Comment (jameerpathan111) [reply tests/foreman/ui/test_oscappolicy.py:128]: @mirekdlugosz I want to check this explicitly.

File: tests/foreman/cli/test_oscap_tailoringfiles.py [hunk 2]
```diff
@@ -206,13 +207,15 @@ def test_positive_download_tailoring_file(self):
 
         :expectedresults: The tailoring file should be downloaded
 
+        BZ: 1857572
+
         :CaseImportance: Critical
         """
         name = gen_string('alphanumeric')
-        file_path = '/tmp/{0}.xml'.format(name)
+        file_path = '/var/{}'.format(self.tailoring_file_path)
```
Comment (devendra104) [inline tests/foreman/cli/test_oscap_tailoringfiles.py:215]:
```suggestion
        file_path = f'/var/{self.tailoring_file_path}'
```

Comment (jameerpathan111) [reply tests/foreman/cli/test_oscap_tailoringfiles.py]: @devendra104 updated  the pr.

File: tests/foreman/cli/test_oscap_tailoringfiles.py [hunk 3]
```diff
@@ -206,15 +207,17 @@ def test_positive_download_tailoring_file(self):
...
         name = gen_string('alphanumeric')
-        file_path = '/tmp/{0}.xml'.format(name)
+        file_path = f'/var/{self.tailoring_file_path}'
         tailoring_file = make_tailoringfile({'name': name, 'scap-file': self.tailoring_file_path})
         assert tailoring_file['name'] == name
-        result = TailoringFiles.download_tailoring_file({'name': name, 'path': '/tmp/'})
+        result = TailoringFiles.download_tailoring_file({'name': name, 'path': '/var/tmp/'})
         assert file_path in result[0]
-        result = ssh.command('find {0}'.format(file_path))
+        result = ssh.command(f'find {file_path}')
```
Comment (devendra104) [inline tests/foreman/cli/test_oscap_tailoringfiles.py:220]: In find command, It is always good to redirect the unwanted error into /dev/null, But I am fine with the current implementation.

Comment (jameerpathan111) [reply tests/foreman/cli/test_oscap_tailoringfiles.py]: @devendra104 updated PR.

**Comment by jyejare** [comment]
@jameerpathan111 ACKed but the Travis is failing, Fix it!

**Comment by devendra104** [comment]
Please update the description of the PR.

**Comment by jameerpathan111** [comment]
thanks everyone. :)

**Pull request merged**